bsp/nuclei/tools/sdk_dist.py

```diff
@@ -5,11 +5,10 @@
 sys.path.append(os.path.join(os.path.dirname(cwd_path), 'rt-thread', 'tools'))
 
 # BSP dist function
-def dist_do_building(BSP_ROOT):
+def dist_do_building(BSP_ROOT, dist_dir):
```

Comment:
Please do not modify other files.

Reply:
Per discussed above, I have to modify like this to make `scons --dist` works, otherwise it will report error. 

![图片](https://user-images.githubusercontent.com/1538922/84332850-962fb180-abc0-11ea-8a49-b532c911b443.png)

Reply:
You can search this function in each files of RT-Thread: 
![image](https://user-images.githubusercontent.com/1241087/84339311-bf583e00-abd0-11ea-8a19-61215c96ea79.png)

If you modify this function, there are about 70 error!